lib/daru/vector.rb

```diff
@@ -1159,6 +1156,12 @@ def guard_sizes!
       end
     end
 
+    def guard_type_check value
```

Comment:
I believe you somehow comment on outdated diff, those methods are already private. Maybe some GitHub bug?..